custom_dc/custom_data.md

```diff
@@ -180,6 +186,16 @@ The first set of parameters only applies to `foo.csv`. The second set of paramet
 
 You must specify the provenance details under `sources`.`provenances`; this field associates one of the provenances defined there to this file.
 
+`observationProperties`
+
+: Optional: Additional information about the observations contained in the CSV file. Currently, four properties are supported:
```

Comment:
Consider `s/about the observations/about each observation`

Reply:
Done.